testcontainers/README.md

```diff
@@ -1 +1,10 @@
-# blog-testcontainers
\ No newline at end of file
+# Testcontainers - Bring Your Integration Tests to a New Level, Blog post
+This is a demo code related to the Novatec blog post ....---Link to be done---. </br> 
+It shows the usage of the Testcontainers in a Spring Boot microservice. <br>
+
+## 1. Import Project
+Import the project in IntelliJ as Gradle project. This project uses 
+Gradle Wrapper, thus Gradle should not be installed locally.
```

Comment:
`thus Gradle should not be installed locally` >> `Gradle does not need to be installed locally`